core/Tracker.php

```diff
@@ -132,8 +132,6 @@ public function trackRequest(Request $request)
         if ($request->isEmptyRequest()) {
             Common::printDebug("The request is empty");
         } else {
-            $this->loadTrackerPlugins();
```

Comment:
This took a long time and it is actually not needed to reload Tracker plugins each time. It was like a year ago when we handled `dp=1` here but this is no longer the case. `dp=1` is now handled in the Provider plugin itself. Saves a lot of performance.